pkgs/by-name/en/encore/tsparser.nix

```
@@ -0,0 +1,76 @@
+{
+  runtimesPath,
+  rustPlatform,
+  makeWrapper,
+  pkg-config,
+  protobuf,
+  openssl,
+  version,
+  src,
+  lib,
+  ...
+}: let
+  cargoLockFile = builtins.path {
+    name = "tsparser-encore-cargo-lock";
+    path = ./tsparser-Cargo.lock;
+  };
+in
+  rustPlatform.buildRustPackage {
+    pname = "tsparser-encore";
+    version = version;
+
+    inherit src;
```

This can be shortened to:
```suggestion
    inherit version src;
```

---

Ups